app/models/current.rb

```diff
@@ -0,0 +1,4 @@
+class Current < ActiveSupport::CurrentAttributes
```

Comment:
@alis-khadka nice! We should use this singleton to replace calls to `Subdomain.current` that cost us a DB round trip